release-notes/azure/v8.4.0.md

```diff
@@ -0,0 +1,21 @@
+## :zap:  Giant Swarm Release 8.4.0 for Azure is now active for you! :zap:
+
+This release contains Kubernetes 1.14.6 and therefore fixes for CVE-2019-9512 and CVE-2019-9514.
+
+### Kubernetes v1.14.6
+- Updated from v1.14.5 - [changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.14.md#v1146)
```

Comment:
```suggestion
- Update from v1.14.5 - [changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.14.md#v1146).
```

Reply:
Currently we use past tense everywhere here and we don't use dots at the end unless it is a proper sentence.

I am up for discussion for changing this in general but then I would provide a separate PR and get it straight everywhere.

Reply:
How about this https://github.com/giantswarm/releases/pull/32/files#diff-53a25c9124eb60134f8a75b498d5d8c5R16-R18 ?

I already talked with @marians a while ago (during specing VOO) and we agreed to use imperative form. We didn't talk about terminating dots but I think it's fine. Especially if there is a point containing more than one sentence.

Reply:
Okay, I will deal with this in a follow up. I merge here now and then align this afterwards to be imperative (want to show this at engineering day).